hphp/hack/src/parser/aast_check.rs

```diff
@@ -78,6 +78,22 @@ impl Checker {
             .push(SyntaxError::make(start_offset, end_offset, msg, vec![]));
     }
 
+    fn add_error_with_quickfix(
+      &mut self,
+      start_offset: usize,
+      end_offset: usize,
+      msg: ErrorMsg,
+      quickfix_title: &str,
+      edits: Vec<(usize, usize, std::string::String)>,
```

Comment:
The `String` type is in the Rust 'prelude', so it's available without needing the full name.

```suggestion
      edits: Vec<(usize, usize, String)>,
```